pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1146,7 +1146,8 @@
         "4.1.1.0": "4.3.0",
         "4.1.1.1": "4.4.1",
         "4.2.0.0": "4.4.0",
-        "4.2.0.1": "4.4.1"
+        "4.2.0.1": "4.4.1",
+        "4.2.0.2": "4.4.2",
```

Comment:
@weshaggard 
Should this be also "4.2.0.2": "4.4.***3***" ??

Reply:
Yes. If you run the `/t:UpdatePackageIndex` from the pkg directory it should update this correctly.

Reply:
Fixed to `"4.2.0.2": "4.4.3"`